### ISSUE TITLE
New VMs cannot access any Google APIs using gcloud

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#498]
```
What steps will reproduce the problem?

Create a VM on my account, try to do something with gcloud

What is the expected output? What do you see instead?

'gcloud compute instances' list should list the instances but instead I see

<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 403 (Forbidden)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{
margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > 
body{background:url(//www.google.com/images
/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 
22px;overflow:hidden}ins{color:#777;text-de
coration:none}a img{border:0}@media screen and 
(max-width:772px){body{background:none;margin-top:0;max-width:none;pad
ding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1
x/googlelogo_color_150x54dp.png) no-r
epeat;margin-left:-5px}@media only screen and 
(min-resolution:192dpi){#logo{background:url(//www.google.com/images/br
anding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 
100%;-moz-border-image:url(//www.google.com/
images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only 
screen and (-webkit-min-device-pixel-rati
o:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlel
ogo_color_150x54dp.png) no-repeat;-we
bkit-background-size:100% 
100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>403.</b> <ins>That’s an error.</ins>
  <p>Your client does not have permission to get URL <code>/0.1/meta-data/service-accounts/default/acquire?scope=http
s%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email%20https%3A%2F%2Fwww.googlea
pis.com%2Fauth%2Fcloud-platform%20htt
ps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin%20https%3A%2F%2Fwww.googl
eapis.com%2Fauth%2Fcompute</code> fro
m this server.  <ins>That’s all we know.</ins>

What is the output of 'gcloud info'?

$ gcloud info
Google Cloud SDK [0.9.80]

Platform: [Linux, x86_64]
Python Version: [2.7.6 (default, Jun 22 2015, 17:58:13)  [GCC 4.8.2]]
Site Packages: [Disabled]

Installation Root: [/usr/lib/google-cloud-sdk]
Installed Components:
  core: [2015.09.25]
  core-nix: [2015.09.25]
  gcloud: [2015.09.25]
  gsutil-nix: [4.15]
  gsutil: [4.15]
  bq: [2.0.18]
  bq-nix: [2.0.18]
System PATH: 
[/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/lo
cal/games]
Cloud SDK on PATH: [False]
WARNING: There are old versions of the Google Cloud Platform tools on your 
system PATH.
  /usr/bin/git-credential-gcloud.sh
  /usr/bin/bq
  /usr/bin/gsutil
  /usr/bin/gcloud
Installation Properties: [/usr/lib/google-cloud-sdk/properties]
User Config Directory: [/home/richardkazuomiller/.config/gcloud]
User Properties: [/home/richardkazuomiller/.config/gcloud/properties]
Current Workspace: [None]
Workspace Config Directory: [None]
Workspace Properties: [None]
Account: [915414242554-compute@developer.gserviceaccount.com]
Project: [richardkazuo-feedproxy]
Current Properties:
  [core]
    project: [richardkazuo-feedproxy]
    account: [915414242554-compute@developer.gserviceaccount.com]
Logs Directory: [/home/richardkazuomiller/.config/gcloud/logs]
Last Log File: 
[/home/richardkazuomiller/.config/gcloud/logs/2016.01.27/01.46.42.376646.log]

Please provide any additional information below.

I've tried using the https://www.googleapis.com/auth/cloud-platform which 
always worked before, and added all the scopes I need individually. I haven't 
changed anything. I tried to deploy today and couldn't pull from my code 
repository. Help!
```

Original issue reported on code.google.com by `richardk...@gmail.com` on 27 Jan 2016 at 1:52

**Comment by GoogleCodeExporter** [comment]
```
Update: tried running "gcloud compute instances list" again after updating the 
Google Cloud SDK and got this error:

$ gcloud compute instances list
ERROR: gcloud crashed (UnicodeDecodeError): 'ascii' codec can't decode byte 
0xe2 in position 1438: ordinal not in ran
ge(128)
```

Original comment by `richardk...@gmail.com` on 27 Jan 2016 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the update. Please post the stack trace from the log that contains 
the UnicodeDecodeError.
```

Original comment by `gsfow...@google.com` on 27 Jan 2016 at 12:45
- Changed state: **NeedsMoreInfo**

**Comment by GoogleCodeExporter** [comment]
```
2016-01-27 12:53:20,449 DEBUG    root            Loaded Command Group: 
['gcloud', 'compute']
2016-01-27 12:53:20,496 DEBUG    root            Loaded Command Group: 
['gcloud', 'compute', 'instances']
2016-01-27 12:53:21,120 DEBUG    root            Loaded Command Group: 
['gcloud', 'compute', 'instances', 'list']
2016-01-27 12:53:21,169 ERROR    ___FILE_ONLY___ BEGIN CRASH STACKTRACE
Traceback (most recent call last):
  File "/usr/local/share/google/google-cloud-sdk/lib/googlecloudsdk/gcloud_main.py", line 169, in main
    gcloud_cli.Execute()
  File "/usr/local/share/google/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 655, in Execute
    print_error=True)
  File "/usr/local/share/google/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 705, in _HandleKnownErr
or
    msg = u'({0}) {1} {2}'.format(command_path_string, unicode(exc),
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1438: 
ordinal not in range(128)
2016-01-27 12:53:21,170 ERROR    root            gcloud crashed 
(UnicodeDecodeError): 'ascii' codec can't decode b
yte 0xe2 in position 1438: ordinal not in range(128)
2016-01-27 12:53:21,170 INFO     ___FILE_ONLY___ 
If you would like to report this issue, please run the following command:
2016-01-27 12:53:21,170 INFO     ___FILE_ONLY___   gcloud feedback

```

Original comment by `richardk...@gmail.com` on 27 Jan 2016 at 12:55

**Comment by GoogleCodeExporter** [comment]
```
Hello, is anyone looking into this? Three days seems like a long time to let 
this issue go unaddressed! I think it goes without saying, but being to create 
new VMs that can use Google Cloud APIs is kind of a core feature. If this 
doesn't get fixed soon I'm going to have to find a different place to host my 
service!
```

Original comment by `richardk...@gmail.com` on 30 Jan 2016 at 2:44

**Comment by GoogleCodeExporter** [comment]
```
I need to get this fixed by Monday morning JST. I didn't choose that date, it 
just has to be done. If it is not resolved, I won't be able to use Google Cloud 
Platform anymore, as sad as that makes me. Someone from technical support told 
me engineers were looking into this but haven't heard anything since. Could I 
please get some kind of status update?
```

Original comment by `richardk...@gmail.com` on 30 Jan 2016 at 3:21

**Comment by GoogleCodeExporter** [comment]
```
Update: Now I'm getting the access denied error from my existing VMs - not just 
the new ones I create. The last log I have of a server being able to 
successfully access Google APIs was at 2016-01-30 12:33:58 GMT.
```

Original comment by `richardk...@gmail.com` on 30 Jan 2016 at 12:51

**Comment by GoogleCodeExporter** [comment]
```
Apologies for the delay on getting back. The unicode output errors should be 
fixed in the latest release 98.0.0.
```

Original comment by `gsfow...@google.com` on 2 Mar 2016 at 7:36
- Changed state: **Fixed**
